builder.toml

```diff
@@ -97,3 +97,4 @@ description = "Ubuntu bionic base image with buildpacks for Java, .NET, NodeJS,
   id = "io.buildpacks.stacks.bionic"
   build-image = "docker.io/paketobuildpacks/build:0.0.79-full-cnb"
   run-image = "index.docker.io/paketobuildpacks/run:full-cnb"
```

Comment:
Unrelated to the diff but just noticing that we use `docker.io` for the `build-image` reference and `index.docker.io` for the run image. It would probably be less confusing if this was consistent.

Reply:
@ekcasey Agree. It's the `docker/distribution` golang library that converts `index.docker.io` to `docker.io` when it finds a new image in the registry. (https://github.com/docker/distribution/blob/v2.7.1/reference/normalize.go#L69-L71) I filed this issue to change all documented stack images to print domain as docker.io https://github.com/paketo-buildpacks/stacks/issues/27